docs/services/navigation/_index.md

```diff
@@ -115,6 +119,56 @@ The following attributes are available for `Navigation` services:
+
+{{% /alert %}}
+
+To make sure your rover base's autonomous GPS navigation with the Navigation Service is accurate, configure and calibrate the Frame System Service for the components of your robot.
+To start, add the Frame System Service to your rover [base](/components/base/) and [movement sensor](/components/movement-sensor/).
+
+- Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+  Scroll to the card with the name of your base:
+
+    {{< imgproc src="/services/navigation/click-add-frame-ui.png" alt="The button to add a frame selected with the cursor on the Viam app config builder." resize="500x" >}}
+
+- Click **Add Frame**.
+
+  - Keep the `parent` frame as `world`.
+  Select the **Geometry** drop-down menu.
+  - Configure a **Geometry** for the base that reflects its physical dimensions.
+  Reference [these instructions](/services/frame-system/#bounding-geometries) to configure your geometry and measure the physical dimensions of your base.
+
+    ![The frame card for the base in the Viam app config builder.](/services/navigation/full-frame-base-ui.png)
+
+- Scroll to the card with the name of your movement sensor.
+  Click **Add Frame** and select the **Parent** box.
+  - Type in the `name` of your base to specify this component as the `parent` of the sensor in the reference frame coordinate system, and click **Save Config** to save your configuration.
+  See [how to configure nested reference frames](/services/frame-system/nested-frame-config/) for an explanation of this configuration process.
+  ![The frame card for the base in the Viam app config builder.](/services/navigation/full-frame-movement-sensor-ui.png)
+
+  - Give the movement sensor a **Translation** that reflects where it is mounted on your base, measuring the coordinates with respect to the origin of the base.
+
+    In other words, designate the origin of the base as `(0,0,0)`, and measure the distance between the origin of the base and the origin of the sensor to obtain the coordinates of the **Translation**.
+
+    See [the Frame System Service](/services/frame-system/#configuration) for more information, and [the Viam Internals](/internals/orientation-vector/) for a detailed guide on conducting this measurement.
+
+Then, to calibrate your frame system for the most accurate autonomous GPS navigation with the Navigation Service:
+
+- After configuring your robot, navigate to the **Control** page and select the card matching the name of your movement sensor.
+- Monitor the readings displayed on the card, and verify that the compass or orientation readings from the movement sensor report `0` when the base is facing north.
+- If you cannot verify this:
+  - Navigate back to your robot's **Config** page.
+  Scroll to the card with the name of your base.
```

Comment:
I think this should be the movement sensor, instead of the base.

Reply:
Amazing catch, thank you Tess

Reply:
That is an important detail indeed

Reply:
I was looking at the doc notes again and I realized that I was confused on the meaning of the sentence outlining that step, but if I was thinking about it the movement sensor has to be the component oriented haha. Guess it's good to spec here